Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -0,0 +1,1516 @@
+        )
+
+    else:
+        return_error(message="Please Enter Job_Id", error='')
+
+
+# Function to execute Fetch Incidents Command
+def fetch_incidents(account_id, client_id, auth_token, Event_list):
+
+    if Event_list == 'Suspicious Activity (EPR)':
+
+        suspicious_activities = get_suspicious_activities(account_id, client_id, auth_token)
+
+        # And retrieve it for use later
+        lastRun = demisto.getLastRun()
+        last_fetch_time = lastRun.get("time")
+        last_count = lastRun.get("count")
+        last_custom = lastRun.get("custom")
+        suspicious_activity_severity = demisto.params().get('suspicious_activity_severity')
+        if suspicious_activity_severity:
+            lastPointer, incident = fetch_suspicious_activities(suspicious_activities, suspicious_activity_severity,
+                                                                last_fetch_time, last_count, last_custom)
+            if lastPointer and incident:
+                demisto.incidents([incident])
+                # You can store the last run time...
+                demisto.setLastRun(lastPointer)
+            else:
+                demisto.incidents([])
+        else:
+            demisto.incidents([])
+
+    elif Event_list == 'RTP Detections (EP)':
+
+        rtp_detections = get_rtp_detections(account_id, client_id, auth_token)
+
+        # And retrieve it for use later
+        lastRun = demisto.getLastRun()
+        last_fetch_time = lastRun.get("time")
+        last_count = lastRun.get("count")
+        last_custom = lastRun.get("custom")
+        rtp_malware_category = demisto.params().get('rtp_threat_category')
+        if rtp_malware_category:
+            category = category_to_code(rtp_malware_category)
+            lastPointer, incident = fetch_rtp_detections(category, rtp_detections,
+                                                         last_fetch_time, last_count, last_custom)
+            if incident and lastPointer:
+                demisto.incidents([incident])
+                # You can store the last run time...
+                demisto.setLastRun(lastPointer)
+            else:
+                demisto.incidents([])
+        else:
+            demisto.incidents([])
+    else:
+        demisto.incidents([])
+
+
+def get_token(client_id, client_secret, account_id):
+    integration_context = demisto.getIntegrationContext()
+    token = integration_context.get('access_token')
+    valid_until = integration_context.get('valid_until')
+    time_now = int(time.time())
+    if token and valid_until:
+        if time_now < valid_until:
+            # Token is still valid - did not expire yet
+            return token
+    # get_nebula_client() should be the implementation of retrieving the token from the API
+    token = get_nebula_client(client_id, client_secret, account_id)
+    integration_context = {
+        'access_token': token,
+        'valid_until': time_now + 3599  # Assuming the expiration time is 1 hour
```

Comment:
does the expiration time not returned in the response from `get_nebula_client` ?
anyway `3599` is too tight, lets change to `3595`